src/controller/event/model/event.go

```diff
@@ -80,3 +85,32 @@ type Scan struct {
 	// ScanType the scan type
 	ScanType string `json:"scan_type,omitempty"`
 }
+
+// CommonEvent ...
+type CommonEvent struct {
+	Operator             string
+	ProjectID            int64
+	OcurrAt              time.Time
+	Operation            string
+	Payload              string
+	SourceIP             string
+	ResourceType         string
+	ResourceName         string
+	OperationDescription string
+	OperationResult      bool
```

Comment:
As if the field used to check whether the operation is success and the type is boolean, so IMO `Result` is not suitable for boolean type, how about use `OperationSucceed`?

Reply:
change to `IsSuccessful`